src/test/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtilTest.java

```diff
@@ -19,6 +33,12 @@
 import org.sagebionetworks.bridge.schema.UploadSchemaKey;
 
 public class BridgeExporterUtilTest {
+    private List<ColumnModel> TEST_COLUMN_MODELS;
+    private List<ColumnDefinition> TEST_COLUMN_DEFINITIONS;
```

Comment:
Minor: non-static member variables should be camel-cased (testColumnModles), not all-caps (TEST_COLUMN_MODELS).

Also, it looks like these aren't being used.